Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -58,27 +55,9 @@
 
 upgradeProperties[2017] = {
     '2017' : {
-        'Geom' : 'DB:Extended',
-        'GT' : 'auto:phase1_2017_realistic',
-        'Era' : 'Run2_2017',
-        'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','HARVESTFull'],
-    },
-    '2017NewFPix' : {
-        'Geom' : 'Extended2017NewFPix',
-        'GT' : 'auto:phase1_2017_realistic',
-        'Era' : 'Run2_2017_NewFPix',
-        'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','HARVESTFull'],
-    },
-    '2017HCALdev' : {
-        'Geom' : 'Extended2017dev',
-        'GT' : 'auto:phase1_2017_hcaldev',
-        'Era' : 'Run2_2017_HCALdev',
-        'ScenToRun' : ['GenSimFull','DigiFull','RecoFull','HARVESTFull'],
-    },
-    '2017AllNew' : {
         'Geom' : 'Extended2017new',
         'GT' : 'auto:phase1_2017_hcaldev',
```

Comment:
what about changes in GT?
Shouldn't the auto-gt name be swapped to reflect the default reference name?
Also, was there any progress with the geometry?